.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Ubuntu CI
+
+on: [push]
+
+jobs:
+  bionic-ci:
+    runs-on: ubuntu-latest
+    name: Ubuntu Bionic CI
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Compile and test
+        id: ci
+        uses: ignition-tooling/action-ignition-ci@master
+        with:
+          apt-dependencies: 'libignition-cmake2-dev libignition-common3-dev libignition-math6-dev libignition-msgs4-dev libignition-rendering2-dev libignition-transport7-dev libignition-tools-dev libignition-plugin-dev libsdformat8-dev'
```

Comment:
We should remove `apt-dependencies` here.

Reply:
removed and added focal

3503a45